androidApp/onbording/src/main/java/tech/takahana/iconwallpaper/android/onbording/ui/screen/WelcomeScreen.kt

```diff
@@ -31,6 +31,9 @@ fun WelcomeScreen() {
         Image(
             painter = painterResource(id = R.drawable.welcome_dog),
             contentDescription = stringResource(R.string.welcome_dog),
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxSize()
```

Comment:
画像の左右の隙間が気になるので、 `contentScale` も指定するといいかも👀
```suggestion
                .fillMaxSize(),
            contentScale = ContentScale.Crop
```

Reply:
修正しました！